bindata/network/ovn-kubernetes/common/008-script-lib.yaml

```
@@ -547,6 +547,11 @@ data:
         multi_network_enabled_flag="--enable-multi-network"
       fi
 
+      network_segmentation_enabled_flag=
+      if [[ "{{.OVN_NETWORK_SEGMENTATION_ENABLE}}" == "true" ]]; then
+        network_segmentation_enabled_flag="--enable-multi-network --enable-network-segmentation"
```

NIT: Can we do this instead to avoid potentially providing `--enable-multi-network` twice?
```suggestion
        multi_network_enabled_flag="--enable-multi-network"
        network_segmentation_enabled_flag="--enable-network-segmentation"
```

---

yeah nice suggestion